BuckarooSdk/Services/VPay/VPayTransaction.cs

```diff
@@ -0,0 +1,32 @@
+﻿using BuckarooSdk.Transaction;
+
+namespace BuckarooSdk.Services.VPay
+{
+  public class VPayTransaction
+  {
+    private ConfiguredTransaction ConfiguredTransaction { get; }
+
+    internal VPayTransaction(ConfiguredTransaction baseTransaction)
+    {
+      ConfiguredTransaction = baseTransaction;
+    }
+
+    public ConfiguredServiceTransaction Pay(VPayPayRequest request)
+    {
+      var parameters = ServiceHelper.CreateServiceParameters(request);
+      var configuredServiceTransaction = new ConfiguredServiceTransaction(ConfiguredTransaction.BaseTransaction);
+      configuredServiceTransaction.BaseTransaction.AddService("VPay", parameters, "pay");
+
+      return configuredServiceTransaction;
+    }
+
+    public ConfiguredServiceTransaction Refund(VPayRefundRequest request)
+    {
+      var parameters = ServiceHelper.CreateServiceParameters(request);
+      var configuredServiceTransaction = new ConfiguredServiceTransaction(ConfiguredTransaction.BaseTransaction);
+      configuredServiceTransaction.BaseTransaction.AddService("VPay", parameters, "refund");
```

Comment:
And here.

Reply:
I follow the pattern as for other implemented by Buckaroo transactions, eg: IdealProcessingTransaction.